src/pages/test-report/test-report.effects.ts

```diff
@@ -138,4 +138,23 @@ export class TestReportEffects {
     delay(1000), // Added a 1 second delay to allow other action to complete/effects to fire
     map(() => new testsActions.PersistTests()),
   );
+
+  @Effect()
+  terminateTestFromTestReport$ = this.actions$.pipe(
+    ofType(
+      testReportActions.TERMINATE_TEST_FROM_TEST_REPORT,
+    ),
+    withLatestFrom(
```

Comment:
Why select the current test if we don't use it for anything?

Reply:
Good find, have updated here: https://github.com/dvsa/mes-mobile-app/pull/564/files#diff-4ccf0d70ae9ceda46a4c611247cd9889R147